csp-guru/datapackage.json

```diff
@@ -460,7 +460,7 @@
     ],
     "licenses": [
         {
-            "name": "CC-BY 4.0",
+            "name": "CC-BY-4.0",
```

Comment:
I'm not convinced scheinbar kann man auch gar keine bindestiche machen.
Aber passt schon.